src/components/EmojiPicker.vue

```diff
@@ -64,18 +64,31 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+@import 'vuetify/settings';
+
+$dark-theme-border-color: rgba(map-get($shades, 'white'), var(--v-border-opacity));
+$light-theme-border-color: rgba(map-get($shades, 'black'), var(--v-border-opacity));
+
 .emoji-picker {
   border-radius: 8px;
 
   em-emoji-picker {
     width: 264px;
     height: 264px;
     border-radius: 8px;
+    border-color: $light-theme-border-color;
+    border-width: 1px;
+    border-style: solid;
 
     #root {
       border-top-left-radius: 0;
       border-top-right-radius: 0;
     }
   }
 }
+.v-theme--dark {
+  em-emoji-picker {
+    border-color: $dark-theme-border-color;
```

Comment:
No need in defining a constant. It is used only in one place. Use the color value directly:

```suggestion
    border-color: rgba(map-get($shades, 'black'), var(--v-border-opacity))
```